tools/sgen/input/FluentUIStyle.yml

```diff
@@ -403,3 +403,59 @@ MSFPrimaryHeaderTokens extends MSFHeaderFooterTokens:
   textColor: $Colors.Foreground.neutral1
   textFont: $Typography.headline
   topPadding: $Spacing.medium
+
+AP_MSFNotificationTokens:
+  backgroundColor: $Colors.Brand.tint40
+  foregroundColor: "FluentUIColor(light: $Colors.Brand.shade10, dark: $Colors.Brand.shade30)"
+  cornerRadius: $Border.radius.xLarge
+  presentationOffSet: $Spacing.small
+  horizontalPadding: 16pt
+  verticalPadding: 12pt
+  verticalPaddingForOneLine: 16pt
+  horizontalSpacing: 16pt
+  minimumHeight: 64.0
+  minimumHeightForOneLine: 56.0
+  shadow1Color: $Shadow.shadow16.color1
```

Comment:
Can we give these more descriptive names other than `shadow1` and `shadow2`? I see they map to properties called `perimeterShadow` and `ambientShadow` in `NotificationView`; maybe those would be good names?